app/components/global/dashboard/selectors.js

```diff
@@ -64,7 +64,7 @@ const teamsSelector = createSelector([teamsListSelector], (teamsList) => {
   if (teamsList) {
     let teamsData = teamsList.toJS();
 
-    ["Run", "Swim", "Walk", "Rowing", "CyclingConverted"].map(
```

Comment:
I think this one will need to stay as-is as we want the Teams tab to still show converted cycling?